src/docs/reference/modules/ContentTypes/README.md

```diff
@@ -186,5 +188,26 @@ public class ProductController : Controller
              Price = productPart.Price.Value,
         });
     }
+    
+    [HttpPost("/api/product/{productId}/price/{price}")]
+    public async Task<ContentValidateResult> UpdateProductPriceAsync(string productId, int price)
+    {
+        var product = _orchardHelper.GetContentItemByIdAsync(productId);
+
+        if (product == null) 
+        {
+            return NotFoundObjectResult();
+        }
+
+        var productPart = product.As<Product>();
+        productPart.Price.Value = price;
+        
+        product.Apply(productPart) //apply modified part to a content item
+        
+        await _contentManager.UpdateAsync(product); //update will fire handlers which could alter the content item.
+
+        //validation will cancel changes if product is not valid. It's fired after update since handlers could change the object.
+        return await _contentManager.ValidateAsync(product);
```

Comment:
This probably needs a publishasync option shown as well, otherwise the item will be left in an unknown state

Reply:
Looking at the code of DefaultContentManager, it basically calls save on _session and a bunch of handlers. So i think what it does it leaves the current state as is (published or draft). I think this is good enough for a sample. I can add a note about publishing though?